package.json

```
@@ -3,39 +3,41 @@
   "name": "app-toolbelt",
   "description": "Toolbelt for digitalfabrik apps",
   "private": true,
-  "type": "commonjs",
+  "type": "module",
   "bin": {
     "app-toolbelt": "bin/app-toolbelt"
   },
   "scripts": {
-    "start": "ts-node src/start.ts",
+    "start": "tsx src/start.ts",
     "build": "tsc --build",
-    "test": "jest",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

:upside_down_face: Maybe this would be simpler, see https://jestjs.io/docs/ecmascript-modules:
```suggestion
    "test": "node --experimental-vm-modules $(yarn bin jest)",
```

---

:x: Some of the tests fail, please check and fix.